### PR TITLE
Add code to run test, get malloc counts

### PR DESCRIPTION
**Pull request by richsalz**
I have to think that getenv hack is a hack and that there are betters way to do this.

### CODE REVIEW

**Comment by richsalz** [comment]
By the way, assuming the counting is correct, holy cow!
```
; OPENSSL_TEST_GETCOUNTS=1 ./test/sslapitest apps/server.pem apps/server.pem 
malloc 59690 realloc 517 free 51127
```

File: test/sslapitest.c [hunk 1]
```diff
@@ -3114,6 +3115,22 @@ int setup_tests(void)
             || !TEST_ptr(privkey = test_get_argument(1)))
         return 0;
 
+    if (getenv("OPENSSL_TEST_GETCOUNTS") != NULL) {
+#ifndef OPENSSL_NO_CRYPTO_MDEBUG
```
Comment (levitte) [inline test/sslapitest.c]: `s|#ifndef|#ifdef|`

**Comment by richsalz** [comment]
@paulidale is there a better way (such as examining the command-line?) to do this?

**Comment by richsalz** [comment]
Ping @paulidale for comment/help.

**Comment by richsalz** [comment]
Lack of a better way, approved so merged. thanks.

**Pull request closed without merging**